tests/vmi_configuration_test.go

```diff
@@ -287,6 +289,30 @@ var _ = Describe("[sig-compute]Configurations", decorators.SigCompute, func() {
 			),
 		)
 
+		It("with downwardMetrics virtio-serial", func() {
+
+			vmi := libvmi.New(
```

Comment:
You might be able to use `libvmi.NewCirros` instead of defining a lot of these options below?

Reply:
Done.